.github/workflows/release.yml

```diff
@@ -22,23 +22,85 @@ jobs:
 
     - name: Build
       run: swift build -v -c release
+    - name: Import Application Siging
+      uses: apple-actions/import-codesign-certs@v1
+      with: 
+        p12-file-base64: ${{ secrets.CERTIFICATES_APPLICATION_P12 }}
+        p12-password: ${{ secrets.CERTIFICATES_APPLICATION_P12_PASSWORD }}
+        create-keychain: true
+        keychain-password: ${{ secrets.APPLE_TEAM_ID }}
+    - name: Import Installer Siging
+      uses: apple-actions/import-codesign-certs@v1
+      with: 
+        p12-file-base64: ${{ secrets.CERTIFICATES_INSTALLER_P12 }}
+        p12-password: ${{ secrets.CERTIFICATES_INSTALLER_P12_PASSWORD }}
+        create-keychain: false
+        keychain-password: ${{ secrets.APPLE_TEAM_ID }}
+    - name: Code Sign
+      run: |
+        codesign --force --options runtime --sign 'Developer ID Application: Mathijs Kadijk (5Z49PA849J)' .build/release/rswift
     - name: Archive
-      run: zip --junk-paths ${{ runner.temp }}/archive.zip .build/release/rswift License
-    - name: Upload
+      run: zip --junk-paths ${{ runner.temp }}/rswift-${{ github.event.release.tag_name }}.zip .build/release/rswift License
+    - name: Package
```

Comment:
Does providing a PKG have advantages? We didn't until now, so I feel maybe we should just keep it at a zip only?